libs/ballot-interpreter/src/hmpb-rust/timing_marks.rs

```diff
@@ -1639,20 +1716,21 @@ pub fn rect_could_be_timing_mark(geometry: &Geometry, rect: &Rect) -> bool {
     let min_timing_mark_width_multiplier = if is_near_left_or_right_edge {
         0.20
     } else {
-        0.75
+        0.5
     };
     let min_timing_mark_height_multiplier = if is_near_top_or_bottom_edge {
         0.20
     } else {
-        0.75
+        0.5
```

Comment:
This allows the timing marks to be smaller when we're doing this initial filter, expanding the set of possible timing marks. I feel okay with this because we still compute another score based on template matching later that should give good results.